src/modules/taskAdd.js

```diff
@@ -109,8 +109,8 @@ class Task {
 }
 
 if ('tasks' in localStorage) {
-  const localData = JSON.parse(localStorage.tasks);
-  localData.forEach((each) => {
```

Comment:
- [ **OPTIONAL** ] In the `README.md` file: I think it would be better if you improve your `Getting Starting section` of your documentation  by adding some important information like how:
   - To access the cloned directory
   - To install dependencies after cloning the repo
   - How to run the server locally
   - Etc.
I can suggest taking a look at this **[Template](https://github.com/tresorsawasawa/react-todo-app)**, even if it isn't the best example, and improve your documentation.